Code/Framework/AzToolsFramework/AzToolsFramework/UI/DocumentPropertyEditor/DocumentPropertyEditor.cpp

```diff
@@ -540,6 +539,12 @@ namespace AzToolsFramework
         }
     }
 
+    void DPERowWidget::SetAttributesFromDom(const AZ::Dom::Value& domArray)
```

Comment:
Does it make sense to put all of the data extracted with `ExtractFromDomNode` here? I see several calls in `DPERowWidget`.